core/src/core-api/navigation.js

```diff
@@ -49,6 +53,88 @@ class LuigiNavigationManager {
     return new linkManager().openAsModal(path, modalSettings);
   }
 
+  /** 
+   * Opens a view in a split view. You can specify the split view's title and size. If you don't specify the title, it is the node label. If there is no node label, the title remains empty. The default size of the split view is 40, which means 40% height of the split view.
+   * @memberof LuigiNavigation
+   * @param {string} path navigation path
+   * @param {Object} splitViewSettings opens a view in a split view. Use these settings to configure the split view's behaviour
+   * @param {string} splitViewSettings.title split view title. By default, it is the node label. If there is no label, it is left empty
+   * @param {number} [splitViewSettings.size=40] height of the split view in percent
+   * @param {boolean} [splitViewSettings.collapsed=false] opens split view in collapsed state
+   * @since NEXTRELEASE
+   * @example
+   * Luigi.navigation().openAsSplitView('projects/pr1/users', {title:'Users', size:'40'});
+   */
+  openAsSplitView(path, splitViewSettings) {
+    return Luigi.splitView.openAsSplitView(path,splitViewSettings);
+  }
+
+  /**
+   * Closes split view opened with openAsSplitView(path, splitViewSettings)
+   * @memberof LuigiNavigation
+   * @since NEXTRELEASE
+   * @example
+   * Luigi.navigation().closeSplitView();
+   */
+  closeSplitView() {
+    return Luigi.splitView.closeSplitView();
+  }
+
+  /**
+   * Collapse split view opened with openAsSplitView(path, splitViewSettings)
```

Comment:
```suggestion
   * Collapses split view opened with openAsSplitView(path, splitViewSettings)
```

For consistency with the other descriptions